salt/kallithea-demo/configs/production.ini

```diff
@@ -0,0 +1,604 @@
+## handling that. Set this variable to 403 to return HTTPForbidden
+auth_ret_code =
+
+## locking return code. When repository is locked return this HTTP code. 2XX
+## codes don't break the transactions while 4XX codes do
+lock_ret_code = 423
+
+## allows to change the repository location in settings page
+allow_repo_location_change = True
+
+## allows to setup custom hooks in settings page
+allow_custom_hooks_settings = True
+
+
+####################################
+###        CELERY CONFIG        ####
+####################################
+
+use_celery = false
+broker.host = localhost
+broker.vhost = kallitheavhost
+broker.port = 5672
+broker.user = kallitheauser
+broker.password = kallitheapass
+
+celery.imports = kallithea.lib.celerylib.tasks
+
+celery.result.backend = amqp
+celery.result.dburi = amqp://
+celery.result.serialier = json
+
+#celery.send.task.error.emails = true
+#celery.amqp.task.result.expires = 18000
+
+celeryd.concurrency = 2
+#celeryd.log.file = celeryd.log
+celeryd.log.level = debug
+celeryd.max.tasks.per.child = 1
+
+## tasks will never be sent to the queue, but executed locally instead.
+celery.always.eager = false
+
+####################################
+###         BEAKER CACHE        ####
+####################################
+
+beaker.cache.data_dir=%(here)s/data/cache/data
+beaker.cache.lock_dir=%(here)s/data/cache/lock
+
+beaker.cache.regions=super_short_term,short_term,long_term,sql_cache_short,sql_cache_med,sql_cache_long
+
+beaker.cache.super_short_term.type=memory
+beaker.cache.super_short_term.expire=10
+beaker.cache.super_short_term.key_length = 256
+
+beaker.cache.short_term.type=memory
+beaker.cache.short_term.expire=60
+beaker.cache.short_term.key_length = 256
+
+beaker.cache.long_term.type=memory
+beaker.cache.long_term.expire=36000
+beaker.cache.long_term.key_length = 256
+
+beaker.cache.sql_cache_short.type=memory
+beaker.cache.sql_cache_short.expire=10
+beaker.cache.sql_cache_short.key_length = 256
+
+beaker.cache.sql_cache_med.type=memory
+beaker.cache.sql_cache_med.expire=360
+beaker.cache.sql_cache_med.key_length = 256
+
+beaker.cache.sql_cache_long.type=file
+beaker.cache.sql_cache_long.expire=3600
+beaker.cache.sql_cache_long.key_length = 256
+
+####################################
+###       BEAKER SESSION        ####
+####################################
+## Type of storage used for the session, current types are
+## dbm, file, memcached, database, and memory.
+## The storage uses the Container API
+## that is also used by the cache system.
+
+## db session ##
+#beaker.session.type = ext:database
+#beaker.session.sa.url = postgresql://postgres:qwe@localhost/kallithea
+#beaker.session.table_name = db_session
+
+## encrypted cookie client side session, good for many instances ##
+#beaker.session.type = cookie
+
+## file based cookies (default) ##
+#beaker.session.type = file
+
+beaker.session.key = kallithea
+beaker.session.secret = {65f405c2-a984-441b-8610-304eec28bec9}
```

Comment:
Does this need to be actually secret? If so it should use pillar data and have a demo key commited inside of ``pillar/dev/secrets/`` and we'll generate a production secret when we deploy it.